lib/source/Instruction.cpp

```diff
@@ -1215,7 +1220,7 @@ spvgentwo::Instruction* spvgentwo::Instruction::opSelect(Instruction* _pCondBool
 
 	if (trueType == nullptr || falseType == nullptr || condType == nullptr || module == nullptr) return error();
 
-	if(*trueType != *falseType)
+	if (*trueType != *falseType)
```

Comment:
Your formatting rules don't really seem to match the existing code, please try to adapt the existing style (not a hard requirement for getting a PR merged, but would be nice)